resolwe_bio/docker_images/base/Dockerfile.ubuntu-14.04

```diff
@@ -0,0 +1,77 @@
+FROM docker.io/resolwe/base:ubuntu-14.04
+
+MAINTAINER Genialis <dev-team@genialis.com>
+
+# Add common build helpers.
+ADD common.sh /var/cache/build/packages-manual/common.sh
+
+# Install base dependencies, which are needed for the common build helpers.
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get --no-install-recommends -y install wget && \
+    apt-get autoremove --purge -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Package manifests and custom build scripts.
+ONBUILD ADD packages-*.txt /var/cache/build/
+ONBUILD ADD packages-manual /var/cache/build/packages-manual/
+
+ONBUILD RUN export DEBIAN_FRONTEND=noninteractive && \
+
+    # Install packages.
+    if [ -f /var/cache/build/packages-debian.txt ]; then \
+        echo "Installing distribution packages..." && \
+        apt-get update && \
+        sed -e 's/^#.*$//g' -e '/^$/d' /var/cache/build/packages-debian.txt | \
+            xargs apt-get --no-install-recommends -y install \
+        ; \
+    fi && \
+
+    # Install Python packages.
+    if [ -f /var/cache/build/packages-python2.txt ]; then \
+        echo "Installing Python 2 packages..." && \
+        pip install --require-hashes -r /var/cache/build/packages-python2.txt \
+        ; \
+    fi && \
+
+    if [ -f /var/cache/build/packages-python2-stage2.txt ]; then \
+        echo "Installing stage 2 Python 2 packages..." && \
+        pip install --require-hashes -r /var/cache/build/packages-python2-stage2.txt \
```

Comment:
Hmm... this seems a bit awkward. Is `pip` really incapable of installing all Python packages with one call?

Reply:
Auch... another of many pip's shortcomings...